gapis/api/templates/api_spy.cpp.tmpl

```diff
@@ -468,4 +469,16 @@
   {{else}}
     {{Template "C++.Statement.Default" $}}
   {{end}}
-{{end}}
\ No newline at end of file
+{{end}}
+
+
+{{/*
+-------------------------------------------------------------------------------
+  Emits the logic to execute a print statement
```

Comment:
I'm trying to understand this PR, and i'm probably failing sometimes. Is this comment accurate? Isn't this generating the print statement, rather than executing it (which happens in the mutate template below)?

Reply:
Correct. It doesn't execute it, it *emits* the code that will execute the print. This is a comment I copy-pasted and adjusted from other templates. For some reason we've used "logic to execute" rather than "code" in all the templates...